src/renderer/component/channelTile/view.jsx

```diff
@@ -75,7 +77,7 @@ class ChannelTile extends React.PureComponent<Props> {
             </React.Fragment>
           )}
           {subscriptionUri && (
-            <div className="card__actions">
+            <div className="media__actions">
               <SubscribeButton uri={subscriptionUri} />
```

Comment:
This is misaligned on the search page
<img width="400" alt="screen shot 2018-12-12 at 3 22 17 pm" src="https://user-images.githubusercontent.com/16882830/49896640-f5710200-fe21-11e8-9aea-a59785d269c0.png">